app/js/components/manageApps/AddonList.jsx

```diff
@@ -38,16 +38,23 @@ export const AddonList = ({handleDownload, install, appList, openPage, openModal
                 className="text-center"
                 id="delete-icon-wrapper"
               >
-                {install == false ? 
-                  <i className="glyphicon glyphicon-trash text-danger delete-icon"  onClick={openModal(app)}/> :
-                  <i className="glyphicon glyphicon-download-alt text-primary install-icon"  onClick={(e) => handleDownload(e)}/> 
+                {
+                  install == false ? 
```

Comment:
I believe you did not write this piece of code, however I would like to suggest we use `strict comparison` i.e (===), as opposed to using `type–converting comparisons`(==) here.

Reply:
Sure thing